src/modals/storage-connections/StorageConnectionsModal.js

```diff
@@ -0,0 +1,154 @@
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+import PluginApiModal from '_/components/modals/PluginApiModal'
+import getPluginApi from '_/plugin-api'
+import { msg } from '_/intl-messages'
+import { Spinner } from 'patternfly-react'
+import { Button } from '@patternfly/react-core'
+import StorageConnectionsModalBody from './StorageConnectionsModalBody'
+import { createErrorMessage } from '_/utils/error-message'
+import './connections.css'
+
+const StorageConnectionsModal = ({
+  isLoading = false,
+  storageDomain,
+  domainConnections,
+  allConnections,
+  onClose = () => { },
+  doConnectionCreate = () => { },
+  doConnectionEdit = () => { },
+  doConnectionDelete = () => { },
+  doConnectionAttach = () => { },
+  doConnectionDetach = () => { },
+  doRefreshConnections = () => { },
+}) => {
+  const [isOpen, setOpen] = useState(true)
+  const [error, setError] = useState(null)
+  const [isShowAll, setShowAll] = useState(false)
+
+  const warning = (!storageDomain || storageDomain.status === 'maintenance')
+    ? null
+    : {
+      title: msg.storageConnectionsDomainNotInMaintenanceWarning(),
+      detail: msg.storageConnectionsDomainNotInMaintenanceWarningDetail(),
+    }
+
+  const close = () => {
+    setOpen(false)
+    onClose()
+  }
+
+  const modalActions = [
+    <Button
+      key='storage-connections-modal-cancel-button'
+      variant='primary'
+      onClick={close}
+    >
+      {msg.closeButton()}
+    </Button>,
+  ]
+
+  const createTitle = () => {
+    let title = msg.storageConnectionsTitle()
+    if (storageDomain) {
+      title = title + ' - ' + storageDomain.name
```

Comment:
avoid concatenating strings in the code - in this case create a parametrized  version `storageConnectionsTitle` label

Reply:
Yes, exactly for loading. I thought that it would be good to have the domain name that we are editing somewhere.
@sjd78 Do you think it's redundant?